web/src/invite.js

```diff
@@ -294,18 +252,18 @@ function open_invite_user_modal(e) {
 
         autosize($("#invitee_emails").trigger("focus"));
 
-        set_custom_time_inputs_visibility(
+        settings_ui.set_custom_time_inputs_visibility(
             custom_expiration_time_unit,
             custom_expiration_time_input,
         );
 
-        const expiration_time_in_minutes = get_expiration_time_in_minutes(
+        const expiration_time_in_minutes = settings_ui.get_expiration_time_in_minutes(
             custom_expiration_time_unit,
             custom_expiration_time_input,
             $("#expires_in"),
         );
         const valid_to_text = valid_to(expiration_time_in_minutes);
-        set_expires_on_text(valid_to_text);
+        settings_ui.set_expires_on_text(valid_to_text);
```

Comment:
I am not sure whether moving these to `settings_ui` is correct since I guess we do not consider invite modal as part of settings UI as per the other classes used for it. But I am not sure what would be the better place though, so we can keep it as it is for now.

Reply:
I think that's probably OK -- it's a generic helper for managing our settings-style inputs; I think of the "invitation" form as a settings modal that happens to not be inside the main settings UI...